tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: horizon.openstack.org/v1beta1
```

Comment:
`v1beta1` needs to be `v1alpha1`, as this is the current version of the `Horizon` API